debugtools/DDR_VM/src/com/ibm/j9ddr/J9DDRClassLoader.java

```diff
@@ -63,32 +62,38 @@ private static boolean shouldGeneratePointerClasses(StructureReader reader) {
 			return false;
 		}
 
-		// Beginning with VM version 29, pointers classes are generated
-		// if "generate.pointers" is true in dynamic.properties.
-
-		String packageName = reader.getPackageName(PackageNameType.POINTER_PACKAGE_SLASH_NAME);
-		String resourceName = '/' + packageName + "dynamic.properties";
-		InputStream stream = J9DDRClassLoader.class.getResourceAsStream(resourceName);
-
-		if (stream == null) {
-			return false;
-		}
-
-		Properties dynamic = new Properties();
-
 		try {
-			dynamic.load(stream);
-		} catch (IOException e) {
+			// Get pointer sizes in bytes (rounding up for 31-bit s390 VMs).
+			int hostPointerSize = (Integer.getInteger("sun.arch.data.model", 0) + 7) / 8;
+			int corePointerSize = reader.getSizeOfUDATA();
+
+			if (hostPointerSize == corePointerSize) {
+				// The VM that produced the core file has the same size pointers
+				// as this VM: we can use the pointers classes in j9ddr.jar.
+				return false;
+			}
+		} catch (SecurityException e) {
```

Comment:
Not sure ignoring a SecurityException is the right answer, as it hides the problem and will continue on to enable pointer generation in cases where it shouldn't.

Reply:
I can return false if an exception is caught if you like.

Reply:
I was thinking to just allow the SecurityException to be thrown, and let the user add a workaround if necessary.